modules/Movecraft/src/main/java/net/countercraft/movecraft/localisation/I18nSupport.java

```diff
@@ -27,21 +27,41 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
 import java.util.Properties;
 import java.util.logging.Level;
 
 public class I18nSupport {
     private static Properties languageFile;
 
-    public static void init() {
-        languageFile = new Properties();
-
+    public static void init(boolean usePropertiesDefaults) {
         File localisationDirectory = new File(Movecraft.getInstance().getDataFolder().getAbsolutePath() + "/localisation");
 
         if (!localisationDirectory.exists()) {
             localisationDirectory.mkdirs();
         }
 
+        Properties defaultProperties = null;
+        if (usePropertiesDefaults) {
+            Properties defaultProperies4DefaultProperties = new Properties();
+            try {
+                defaultProperies4DefaultProperties.load(new InputStreamReader(
+                        I18nSupport.class.getResourceAsStream("/localisation/movecraftlang_en.properties"),
+                        StandardCharsets.UTF_8));
+                defaultProperties = new Properties(defaultProperies4DefaultProperties);
+                defaultProperties.load(new InputStreamReader(
+                        new FileInputStream(localisationDirectory.getAbsolutePath() + "/movecraftlang_en.properties"),
+                        StandardCharsets.UTF_8));
+            } catch (FileNotFoundException e) {
+                defaultProperties = defaultProperies4DefaultProperties;
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+
+        languageFile = new Properties(defaultProperties);
+
```

Comment:
This would also make it easy for people to make their own custom localization only changing a few things, like if they wanted to change some wording to be more in theme with an airship pirates server...